content/locations/states.yml

```diff
@@ -0,0 +1,299 @@
+state: Alabama
```

Comment:
@nicoleslaw I think for this to be valid yaml, it needs to all be in the same list. For example:
```
states:
  Alabama:
  - url: http://www.acjic.alabama.gov/
    text: Alabama Criminal Justice Information Center
  - url:
    text:

  Alaska:
  - url: http://www.dps.alaska.gov/statewide/ucr.aspx
    text: Alaska Department of Public Safety
  - url:
    text:
...
```

If that is cool with you, I can commit my changes